acmw-db-app/pages/signin.js

```diff
@@ -96,24 +103,43 @@ const TextInput = (props) => {
         onFocus={() => setActive(true)}
         onBlur={() => setActive(false)}
       />
-      <label htmlFor={id} className={error && styles.error}>
-        {error || label}
-      </label>
     </div>
   );
 }
 
 class SignInButton extends React.Component {
```

Comment:
Oh shoot I think I just changed this to using hooks (don't think I merged the ADA-20 yet), but it should be pretty easy to adapt. I think it should be more or less the same...

Reply:
Once ADA-20 is merged I can pull those changes and hopefully that will resolve itself

Reply:
Actually I shouldn't even need to pull unless there are conflicts so I'm asking for approval now